erdpy_network/utils.py

```diff
@@ -0,0 +1,2 @@
+def decimal_to_padded_hex(i):
+    return "0" + f"{i:x}" if len(f"{i:x}") % 2 else f"{i:x}"
```

Comment:
```
as_hex = f"{i:x}"
as_hex_padded = "0" + as_hex if ...
```

(that is, suggestion to extract a temporary variable `as_hex`)

Reply:
Done